### DISCUSSION TITLE
[FR]: CLI version for rooted user

### DISCUSSION THREAD

**Discussion by xDoge26** [lihenggui/blocker#441] [Ideas]
### Is there an existing issue for this?

- [X] I have searched the existing issues

##### Describe the problem

I'm afraid this app will be like myandroidtools when no one maintains it anymore

##### Describe the solution

CLI version will be independent of android api levels

**Comment by lihenggui** [comment]
Can you describe more?

**Comment by xDoge26** [comment]
I mean something like busybox that can be executed with root permission using terminal emulator like [Termux](https://github.com/termux/termux-app)

**Comment by lihenggui** [comment]
You mean a CLI tools that can control IFW, is that correct?

**Comment by xDoge26** [comment]
yes, a cli tool that can control pm,ifw .Plus cli version allow us to use it in TWRP just in case something go wrong

**Comment by lihenggui** [comment]
Previously I decided to make a flashable zip file to do the restoration. I
can consider to do that, but I recently I have limited time in the Blocker
project. Contribution welcomed.
